pkg/workceptor/kubernetes.go

```diff
@@ -346,7 +365,7 @@ mainLoop:
 				retryGetPod,
 				err,
 			)
-			time.Sleep(time.Second * time.Duration(curPodDelay))
+			time.Sleep(time.Duration(int64(kw.GetKubeTimeoutStart()) * int64(curPodDelay)))
```

Comment:
May need some overflow protection here. With `RECEPTOR_KUBE_TIMEOUT_START` set to 1 (the default value), setting `RECEPTOR_KUBE_RETRY_COUNT=25` will result in a 20 hour sleep time. Setting `RECEPTOR_KUBE_RETRY_COUNT=50` will cause a duration overflow value of `-1627909h 44m 8.709551616s`